src/ts/components/dynamicsettingspanelitem.ts

```diff
@@ -91,7 +91,7 @@ export class DynamicSettingsPanelItem extends SettingsPanelItem<DynamicSettingsP
       let selectedOptionLabelText = selectedItem.label;
       if (this.settingComponent instanceof SubtitleSelectBox) {
         let availableSettings = this.settingComponent.getItems().length;
-        selectedOptionLabelText = selectedOptionLabelText + ' (' + (availableSettings - 1) + ')';
+        selectedOptionLabelText = i18n.performLocalization(selectedOptionLabelText) + ' (' + (availableSettings - 1) + ')';
```

Comment:
This was a linter error as the + operator is not allowed on the `LocalizableText` type. This is actually a bug fix.